robot/description/urdf/constants.xacro

```diff
@@ -81,17 +87,30 @@
     <xacro:property name="laser_length" value="0.05"/>
     <xacro:property name="laser_width" value="0.05"/>
     <xacro:property name="laser_height" value="0.07"/>
-    <xacro:property name="laser_mass" value="0.16"/> 
+    <xacro:property name="laser_mass" value="0.16"/>
 
     <xacro:property name="laser_x_dist" value="${base_length/2 - 2*laser_length}"/>
     <xacro:property name="laser_z_dist" value="${laser_height/2}"/>
 
+    <!-- ZED camera -->
+    <!-- NOTE: approx as a rectangular prism -->
+    <xacro:property name="ZED_camera_mass" value="0.1587573"/>
+    <xacro:property name="ZED_camera_length" value="0.175"/>
+    <xacro:property name="ZED_camera_width" value="0.03"/>
+    <xacro:property name="ZED_camera_height" value="0.033"/>
+    <!-- ZED image height/width corresponds to image size for 60Hz refresh-->
+    <xacro:property name="ZED_image_refresh" value="60"/>
+    <xacro:property name="ZED_image_width" value="3840"/>
+    <xacro:property name="ZED_image_height" value="1080"/>
```

Comment:
I may have forgotten to mention this but the image we get from ZED is 720 p (which I believe corresponds to the 30 Hz option)

Reply:
so 720p corresponds to 60hz, and previously, you mentioned it was 30Hz (thus why I changed to these values). maybe, we can stick with this, and add a comment saying check again with real camera if we are unsure?

Reply:
Sure